src/address.spec.ts

```diff
@@ -67,4 +67,33 @@ describe("test address", () => {
             Address.fromBech32("erd1qqqqqqqqqqqqqpgqxwakt2g7u9atsnr03gqcgmhcv38pt7mkd94q6shuwt").isSmartContract(),
         );
     });
+
+    it("should contract address", () => {
+        const addressComputer = new AddressComputer();
+        const deployer = Address.fromBech32("erd1j0hxzs7dcyxw08c4k2nv9tfcaxmqy8rj59meq505w92064x0h40qcxh3ap");
+
+        let contractAddress = addressComputer.computeContractAddress(deployer, 0n);
+        assert.equal(contractAddress.toHex(), "00000000000000000500bb652200ed1f994200ab6699462cab4b1af7b11ebd5e");
+        assert.equal(contractAddress.toBech32(), "erd1qqqqqqqqqqqqqpgqhdjjyq8dr7v5yq9tv6v5vt9tfvd00vg7h40q6779zn");
+
+        contractAddress = addressComputer.computeContractAddress(deployer, 1n);
+        assert.equal(contractAddress.toHex(), "000000000000000005006e4f90488e27342f9a46e1809452c85ee7186566bd5e");
+        assert.equal(contractAddress.toBech32(), "erd1qqqqqqqqqqqqqpgqde8eqjywyu6zlxjxuxqfg5kgtmn3setxh40qen8egy");
```

Comment:
:+1: 

For reviewers: https://github.com/multiversx/mx-sdk-py-core/blob/main/multiversx_sdk_core/address_test.py